components/automate-chef-io/content/docs/event-feed.md

```diff
@@ -7,42 +7,34 @@ bref = ""
 toc = true
 +++
 
-## About the Event Feed
+## Overview
 
 Use the event feed for actionable insights and operational visibility.
-The guitar strings and improved query language help you drill into infrastructure & compliance automation events quickly to isolate errors.
+The guitar strings help you drill into infrastructure & compliance automation events quickly to isolate errors.
+
+The event feed shows the time of the event, its type, the object acted on, the action, and the initiating action.
 
 Users require permission for the `event:*` action to view and interact with *Event Feed*. Filter and search ability in *Event Feed* requires user permission for the `infra:nodes:list` action.
 
-## Event Guitar Strings
+![Event feed](/images/docs/event-feed.png)
```

Comment:
@susanev Why is event-feed.png cited here when it's deleted in this PR?

Reply:
its only 28% deleted, thats what it shows when you update the img

Reply:
![event-feed-change](https://user-images.githubusercontent.com/5489125/83303188-76a48c00-a1b1-11ea-906b-6083821b375c.png)

Reply:
Ah ha! Okay. Now I get it. Thank you for confirming. Will have additional changes shortly